build-docker-image.ps1

```diff
@@ -19,15 +19,15 @@ function Exec
 #Select the UI version from dependencies.props and use it as image version
 
 
-$version = select-xml -Path .\build\dependencies.props -XPath "/Project/PropertyGroup[contains(@Label,'Health Checks Package Versions')]/HealthCheckUI"
```

Comment:
Powershell supports combining `\` and `/` in a single path for Windows, but only `/` is supported in Linux.